composer.json

```diff
@@ -34,7 +34,7 @@
     "psr/log": "^1.1",
     "ramsey/uuid": "^3.9|^4.1.1",
     "monolog/monolog": "^1.25|^2",
-    "kevinrob/guzzle-cache-middleware": "^3.2",
+    "kevinrob/guzzle-cache-middleware": "^4.0",
```

Comment:
I see https://github.com/Kevinrob/guzzle-cache-middleware/pull/157 is part of this major package release. Have you specifically checked what kind of requests Rollout is making / if this breaks caching? Just want to be extra sure cause our frontend api interceptor cache thing does a similar technique where `POST` busts the cache and you have to manually specify which POST endpoints are cacheable.